File: 26.RemoveDuplicatesFromSortedArray/Solution.cpp
#include<bits/stdc++.h>
using namespace std;
class Solution{
    int removeDuplicates(vector<int>& nums){
        int sp = 0, fp = 0;
        while(fp < nums.size()){
            if(nums[sp] != nums[fp]){
                nums[++sp] = nums[fp];
            }
            fp++;
        }
        return sp + 1;
    }
};

File: 26.RemoveDuplicatesFromSortedArray/Solution.java
public class Solution{
    public int removeDuplicates(int[] nums) {
        int sp = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[sp] != nums[i]) {
                nums[++sp] = nums[i];
            }
        }
        return ++sp;
    }
}